build_tools/github_actions/upload_build_artifacts.py

```diff
@@ -116,6 +160,19 @@ def main(argv):
         help="Path to the build directory of TheRock",
     )
 
+    parser.add_argument(
+        "--report-path",
+        type=Path,
+        help="Directory containing .html files to upload (optional)",
+    )
```

Comment:
Per my prior comment, have the tests write their result files into the build or logs directories. No searching other paths here.